spec/stat_tracker_spec.rb

```diff
@@ -136,6 +136,18 @@
       expect(stat_tracker.total_home_goals).to be_a(Hash)
     end
 
+    it "#avg_goals_results" do
```

Comment:
we could maybe try to use a mock here, but not crazy necessary, we can revisit this if we have time.